<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>491</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nIKLabel">
      <property name="text">
       <string>NIK</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editNIK"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="nAMALabel">
      <property name="text">
       <string>NAMA</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editNama"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="hARTALabel">
      <property name="text">
       <string>HARTA</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editHarta"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="totalLabel">
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editTotal"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tabelZakat">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>511</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NIK</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NAMA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TOTAL</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>240</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>UBAH</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
